build-constraints.yaml

```diff
@@ -1452,6 +1452,9 @@ packages:
         - traverse-with-class
         - tuples-homogenous-h98
 
+    "Owen Shepherd <owen+stackage@owen.cafe> @414owen":
+        - tasty-html
+
```

Comment:
@414owen :  I think you need to _move_ `tasty-html` to your section, i.e., also delete it above.

Reply:
Oh really? I was thinking of being `a` maintainer, rather than `the` maintainer.

I guess either works.